default/AI/freeorion_debug/timers.py

```diff
@@ -1,20 +1,32 @@
 import os
 import freeOrionAIInterface as fo
 from time import time
-from option_tools import get_option_dict, check_bool, TIMERS_TO_FILE, TIMERS_USE_TIMERS, TIMERS_DUMP_FOLDER
+from option_tools import get_option_dict, check_bool, _get_default_file_path
+from option_tools import TIMERS_TO_FILE, TIMERS_USE_TIMERS, TIMERS_DUMP_FOLDER
 
 # setup module
 options = get_option_dict()
 
 USE_TIMERS = check_bool(options[TIMERS_USE_TIMERS])
 DUMP_TO_FILE = check_bool(options[TIMERS_TO_FILE])
-TIMERS_DIR = options[TIMERS_DUMP_FOLDER]
+TIMERS_DIR = os.path.join(_get_default_file_path(), options[TIMERS_DUMP_FOLDER])
 
 
 def make_header(*args):
     return ['%-8s ' % x for x in args]
 
 
+def _get_timers_dir():
+    try:
+        if os.path.isdir(fo.getUserDir()) and not os.path.isdir(TIMERS_DIR):
+            os.makedirs(TIMERS_DIR)
+    except OSError:
+        print ("AI Config Error: could not create path %s" % TIMERS_DIR)
+        return fo.getUserDir()
```

Comment:
In case of error in timers, lets prevent writing it somethere.  If user turn on timers an did not found them, he will check log and report an issue.